tests/unit/envs/real_robots/wrappers_test.py

```diff
@@ -0,0 +1,43 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
+#  Copyright (C) 2021, Numenta, Inc.  Unless you have an agreement
+#  with Numenta, Inc., for a separate license for this software code, the
+#  following terms and conditions apply:
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Affero Public License version 3 as
+#  published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#  See the GNU Affero Public License for more details.
+#
+#  You should have received a copy of the GNU Affero Public License
+#  along with this program.  If not, see http://www.gnu.org/licenses.
+#
+#  http://numenta.org/licenses/
+#
+
+import unittest
+
+from gym import spaces
+from real_robots.envs import REALRobotEnv
+
+from nupic.embodied.envs.real_robots import WrapRobot
+
+
+class WrapRobotTest(unittest.TestCase):
+    def setUp(self):
+        pass
```

Comment:
In setUp is where you instantiate any classes or assign attributes that will be shared across methods.

My take on this is that it is better to have a lean setUp and try to initialize as much as possible in each method, sometimes reusing variables across tests can lead to bugs which are hard to track. 

But some things make sense to reuse, like constants, or if something takes a long time to instantiate increasing the test run time significantly.